Gemfile

```diff
@@ -4,8 +4,8 @@ gem 'addressable'
 gem 'gds-api-adapters', '~> 51.2.0'
 gem 'govuk_app_config', '~> 0.3.0'
 gem 'govuk_frontend_toolkit', '~> 7.2.0'
-gem 'govuk_navigation_helpers', '~> 8.2.0'
-gem 'govuk_publishing_components', '~> 4.1.0'
+gem 'govuk_navigation_helpers', '~> 9.0'
+gem 'govuk_publishing_components', '~> 5.0'
```

Comment:
Same comment as before, I'm assuming 5.0 is okay and it doesn't need to be 5.0.0?

Reply:
👍